README.md

```diff
@@ -18,18 +22,22 @@ can check out the tag `v0.2` of this library.
 If you want to use Agda 2.6.0.1 instead of the latest release version, you
 can check out the tag `v0.1` of this library.
 
+Learning materials
```

Comment:
We should add the HoTTest summer school material here: https://github.com/martinescardo/HoTTEST-Summer-School/tree/main/Agda/Cubical and https://www.youtube.com/channel/UC-9jDbJ-HegCFuWuam1SfvQ

Reply:
Added